fms/models/hf/__init__.py

```diff
@@ -1,3 +1,4 @@
+# type: ignore
```

Comment:
this was the fix for mypy?

Reply:
Yes, this was required when i referred to hf in __init__ for get_model